cmd/podman/restart.go

```diff
@@ -47,6 +51,10 @@ func restartCmd(c *cli.Context) error {
 		restartContainers []*libpod.Container
 	)
 
+	if os.Geteuid() != 0 {
```

Comment:
Just curious, why didn't you use rootless.IsRootless() here?

Reply:
IsRootless reports true also when running in the user namespace and running with euid=0, so we want really to check that it is the first part of the rootless setup, when we don't run in a new user namespace

Reply:
Thx for the 411 @giuseppe